src/ScottLogic.Internal.Training/ScottLogic.Internal.Training.Matcher.Tests/MatcherTests.cs

```diff
@@ -290,4 +288,51 @@ public void AddBuyOrder_BuyLess_PartialMatch_NewTrade()
             Assert.Equal(currentTrade, returnedTrade);
         }
     }
+
+    public class ReturnExistingTrades
+    {
+        [Fact]
+        public void NoExistingTrades()
+        {
+            var currentMatcher = new OrderMatcher();
+            
+            Assert.Empty(currentMatcher.ExistingTrades);
+        }
+
+        [Fact]
+        public void OneExistingTrade()
+        {
+            var currentMatcher = new OrderMatcher();
+            var currentOrder1 = new Order(1001, 65, 55, OrderType.Sell, 14);
+            var currentOrder2 = new Order(1002, 55, 55, OrderType.Buy, 15);
+            currentMatcher.ProcessOrder(currentOrder1);
+            currentMatcher.ProcessOrder(currentOrder2);
+
+            var trades = new List<Trade>() { currentMatcher.CurrentTrade };
```

Comment:
I think you should be able to do:

var trades = new List<Trade> { currentMatcher.CurrentTrade };

i.e. remove the () before the {